sojo-api/app/seeds/mongodb/$default.js

```diff
@@ -21,38 +21,39 @@ module.exports = {
       client_secret: process.env.CLIENT_SECRET,
       email: clientName + '@no-reply.com',
       scope: scopes[i],
-      type: 'native'
+      type: 'native',
+      recaptcha_secret: '6LcabzgUAAAAAFy7kjuKtNhA1nuMgcTgQnalVfCR'
```

Comment:
Please Generate a new recaptcha_secret and set the above as an environment variable like so:
```Node
process.env.RECAPTCHA_SECRET
```
so that the key is not part of the git history.